rpcs3/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ else()
 	set(ADDITIONAL_LIBS "")
 endif()
 
+if(UNIX)
```

Comment:
Looking more, those X11 includes are either cruft or a workaround for one of the dependencies (e.g. WxWidgets). So, I've excluded linking against libX11 on OS X in the new version of the commit.